src/components/GisidaWrapper/index.tsx

```diff
@@ -279,6 +279,9 @@ class GisidaWrapper extends React.Component<GisidaProps, GisidaState> {
       nextProps.currentGoal !== this.props.currentGoal &&
       (this.state.locations || this.state.doInitMap)
     ) {
+      if (document.querySelectorAll(MAPBOXGL_POPUP).length) {
+        document.querySelectorAll(MAPBOXGL_POPUP).forEach(el => el.remove());
+      }
```

Comment:
Is there a way we could test this?

Reply:
@cKellyDesign yeah this will be covered in issue #792. simulating a point click on the map using enzyme/react-testing library is what I haven't figured yet.